README.md

```diff
@@ -47,6 +47,7 @@ To view which version of Nutils is currently installed, run:
 
     python3 -m pip show nutils
 
+Nutils can be installed in a Windows machine using WSL environment. If you want to assemble matrices in parallel using nutils on a Windows machine, then WSL is the way to go. Instructions to setup WSL are available [here](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-10#1-overview). After setting up WSL, nutils can be installed using the above instructions. 
```

Comment:
The linked tutorial mentions an [updated version](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support#1-overview) that also works for Windows 11. Can you verify that the updated tutorial works as well? If so, please update the link.

Reply:
The updated tutorial works as well.

Reply:
The updated tutorial works as well. I will update the link.